content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,476 @@
+title: NeuVector Prime in Azure: Usage Instructions
+---
+theme: suse
+---
+body:
+
+##  How to install NeuVector and configure Billing-adapter
+
+The NeuVector offer handles billing by gathering usage data from NeuVector. For complete installation, Ensure to follow all the steps
+till NeuVector Usage Record and Billing.
+
+##Prerequisites
+
+These instructions assume you have an AKS cluster running as per the [NeuVector System requirements document](https://open-docs.neuvector.com/basics/requirements)
+
+
+A step by step walk-through for creating a new deployment of NeuVector from the Azure Marketplace Overview page.
+
+1. Choose the plan from the dropdown list. ( "Plans + Pricing" tab more details about Software plan)
+2. Click 'Create' 
+
+## Basics
+
+1. Select existing subscription from the dropdown list.
+2. Select existing Resource group from the dropdown list.
+3. Select existing AKS Cluster Name from the dropdown list.
+4. Choose name for Cluster extension resource name as it suggested by the 
+   tool tip "Allow alphanumeric, and dots. And the value must be 1-253 characters long". 
+
+
+   !["Existng Resource Group"](./basicclusternv.png)
+
+
+
+
+   ---
+   **NOTE**
+
+   The ***Create new *** Resource group feature is not supported
+
+   ---
+
+
+
+   !["New Resource Group"](./noresourcegroupver.png)
+
+
+
+   click 'Next'
+
+
+
+## NeuVector Configuration
+
+
+1. Choose Admin password as it suggested by the tip
+
+
+!["Application Details"](./adminpassword.png)
+
+
+click 'Next'
+
+## Review + create
+
+
+  It will summarize the offer and link to "view automation template" (Azure Resource Manager Template)
+
+
+     Price
+     Basics
+     NeuVector Configuration
+
+click 'Create'
+
+## Deployment Complete
+
+Deployment will be in progress. After it is completed, the following resource are created
+
+1. AKS Cluster
+2. Kubernetes Service Extension
+
+click 'Go to Resource group'
+click AKS cluster (created by offer)
+
+click 'Connect' Follow Cloud Shell instructions to Set cluster context
+  1. Set the cluster subscription
+  2. Download cluster credentials
+
+
+## Install Scanner Updater
+
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry. It is recommended to install scanner updater AFTER successful initial NeuVector deployment.
```

Comment:
It is *required* to install the scanner after a successful initial NeuVector deployment.